File: src/test/java/com/yah/PersonTest.java
package com.yah;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {
    @Test
    public void bmiTest(){
        Person person = new Person(73.9f,1.69f);
        Assertions.assertEquals(73.9f/(1.69f*1.69f) , person.bmi());
    }
}

File: src/test/kotlin/com/yah/PersonTest.kt
package com.yah.kotlin

import com.yah.Human
import com.yah.Person
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest {
    @Test
    fun humanBmiTest(){
        val human = Human("Nian",55f, 1.63f)
        Assertions.assertEquals(55f/(1.63f*1.63f) , human.bmi())
    }

    @Test
    fun personBmiTest(){
        val person = Person(100.7f,1.87f)
        Assertions.assertEquals(100.7f/(1.87f*1.87f), person.bmi())
    }
}